src/cred-events.c

```diff
@@ -62,7 +62,7 @@ static __always_inline int dispatch_credentials_event(struct pt_regs *__ctx)
     {
         bpf_perf_event_output(__ctx,
                               &cred_events,
-                              bpf_get_smp_processor_id(),
+                              BPF_F_CURRENT_CPU,
```

Comment:
I want to say this was a holdover from when we wanted to support kernels as old as 4.4. The flag is fewer instructions, so this is a nice win.